apps/web/src/pages/increase/[[...currency]].tsx

```diff
@@ -16,37 +16,34 @@ const IncreaseLiquidityPage = () => {
 
   const native = useNativeCurrency()
 
-  const [currencyIdA, currencyIdB] = router.query.currency || [
-    native.symbol,
-    (chainId && CAKE[chainId]?.address) ?? (chainId && USDC[chainId]?.address),
-  ]
+  const [currencyIdA, currencyIdB] =
+    !router.isReady || !chainId
+      ? [undefined, undefined]
+      : router.query.currency || [native.symbol, CAKE[chainId]?.address ?? USDC[chainId]?.address]
 
-  useEffect(() => {
```

Comment:
taken the logic out of useEffect to prevent crash when router is ready and first rerender triggered before useEffect